mesh/mesh.cpp

```diff
@@ -6585,21 +6573,14 @@ void Mesh::GetBdrElementFace(int i, int *f, int *o) const
    }
 }
 
-int Mesh::GetBdrElementEdgeIndex(int i) const
+int Mesh::GetBdrElementFaceIndex(int i) const
```

Comment:
Given this is now so simple, can possibly be inline in the header.

Reply:
Done in c794dae53b1d3723a4d1b0a39aa576d848a2a79e